src/selection.ts

```diff
@@ -57,6 +57,16 @@ export interface BaseSelectionDef {
 }
 
 export interface SingleSelectionConfig extends BaseSelectionDef {
+  /**
+   * Controls clearing selections. Can be `true`, `false`, or a `string` that is
+   * one of the [Supported Event Types](https://vega.github.io/vega/docs/event-streams/).
+   *
+   * __Default value:__ `true`, which corresponds to `dblclick` (i.e.
+   * user doubleclick clears current selection).
+   *
+   * See the [TODO: clear] documentation for more information.
+   */
+  clear?: string | boolean;
```

Comment:
Let's set `clear` to be an `EventStream | boolean`. Even though `EventStream` is aliased to `any`, we prefer more specific typings wherever possible both as a self-documenting mechanism and for when we make the types more specific in the future.

Reply:
Should `EventStream` be `string` or `any`?